lib/util/meta.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var META_SYMBOL = Symbol('json-schema-ref-parser meta');
+
+exports.META_SYMBOL = META_SYMBOL;
```

Comment:
Is there any reason to export this symbol?  I like the idea of it being purely encapsulated in this module, which ensures that people use the `setMetadata()` and `getMetadata()` functions rather than accessing the metadata directly.